topics/homework/week2/task_1.py

```diff
@@ -0,0 +1,66 @@
+#Bubble Sort Algorithm
+#In ascending order (regular one)
+user_input = input("Введите последовательность чисел, разделенных пробелами: ")
+numbers = [int(x) for x in user_input.split()]
+
+def bubble_sort(arr):
+    n = len(arr)
+
+    for i in range(n):
+        for j in range(0, n - i - 1):
+            if arr[j] > arr[j + 1]:
+                arr[j], arr[j + 1] = arr[j + 1], arr[j]
+
+    return arr
+
+sorted_numbers = bubble_sort(numbers)
+
+
+print("Отсортированный по возрастанию список:", sorted_numbers)
```

Comment:
The instruction and some comments below are in Russian. It is convenient for non-Russian speakers.

Reply:
Translated to English